tests/test_validators.py

```diff
@@ -32,7 +32,7 @@
 
 
 def test_annotated_validator_after() -> None:
-    MyInt = Annotated[int, AfterValidator(lambda x: x if x != -1 else 0)]
+    MyInt = Annotated[int, AfterValidator(lambda x, _info: x if x != -1 else 0)]
```

Comment:
This change here covers [this branch in AfterValidator.__get_pydantic_core_schema__](https://github.com/pydantic/pydantic/blob/cf48b1a5a494604ec161ed85df6285d683084baa/pydantic/functional_validators.py#L32).